documentation/staging/content/managing-domains/manage-domains.md

```diff
@@ -124,127 +123,37 @@ You can modify the WebLogic domain configuration for Domain in PV, Domain in Ima
+For Model in Image, you use [Runtime Updates]({{<relref "/managing-domains/model-in-image/runtime-updates.md" >}}).
 
 ### About the Domain resource
 
-For more information, see [Domain resource]({{< relref "/userguide/managing-domains/domain-resource/_index.md" >}}).
+For more information, see [Domain resource]({{< relref "/managing-domains/domain-resource/_index.md" >}}).
 
 ### Managing lifecycle operations
 
 You can perform lifecycle operations on WebLogic Servers, clusters, or domains.
-See [Starting and stopping]({{< relref "/userguide/managing-domains/domain-lifecycle/startup.md" >}}) and [Restarting]({{< relref "/userguide/managing-domains/domain-lifecycle/restarting.md" >}}) servers.
+See [Starting and stopping]({{< relref "/managing-domains/domain-lifecycle/startup.md" >}}) and [Restarting]({{< relref "/managing-domains/domain-lifecycle/restarting.md" >}}) servers.
 
 ### Scaling clusters
 
 The operator lets you initiate scaling of clusters in various ways:
 
-* [Using kubectl to edit the Domain resource]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#on-demand-updating-the-domain-directly" >}})
-* [Using the operator's REST APIs]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#calling-the-operators-rest-scale-api" >}})
-* [Using WLDF policies]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#using-a-wldf-policy-rule-and-script-action-to-call-the-operators-rest-scale-api" >}})
-* [Using a Prometheus action]({{< relref "/userguide/managing-domains/domain-lifecycle/scaling#using-a-prometheus-alert-action-to-call-the-operators-rest-scale-api" >}})
+* [Using kubectl to edit the Domain resource]({{< relref "/managing-domains/domain-lifecycle/scaling#on-demand-updating-the-domain-directly" >}})
+* [Using the operator's REST APIs]({{< relref "/managing-domains/domain-lifecycle/scaling#calling-the-operators-rest-scale-api" >}})
+* [Using WLDF policies]({{< relref "/managing-domains/domain-lifecycle/scaling#using-a-wldf-policy-rule-and-script-action-to-call-the-operators-rest-scale-api" >}})
+* [Using a Prometheus action]({{< relref "/managing-domains/domain-lifecycle/scaling#using-a-prometheus-alert-action-to-call-the-operators-rest-scale-api" >}})
 
 ### About domain events
 
 The operator generates Kubernetes events at key points during domain processing.
-For more information, see [Domain events]({{< relref "/userguide/managing-domains/domain-events.md" >}}).
+For more information, see [Domain events]({{< relref "/managing-domains/accessing-the-domain/domain-events.md" >}}).
 
 ### Monitoring a domain
 
 The operator can export Prometheus-compatible metrics by embedding a WebLogic Monitoring Exporter configuration in its domain specification. For more details, see the WebLogic Monitoring Exporter document, [Use the Monitoring Exporter with WebLogic Kubernetes Operator](https://github.com/oracle/weblogic-monitoring-exporter#use-the-monitoring-exporter-with-weblogic-kubernetes-operator).
```

Comment:
Change title to Monitor and access.  Prefix with current paragraph sentencelike:  "To access the domain using WLST, console, T3, or a load balancer, or to  export Prometheus-compatible metrics, see Monitor and access."  

Add a very small chapter to "Monitor and access" named "Monitoring exporter" that contains the current material.

Reply:
Done.